microsoft-edge/webview2/concepts/multi-profile-support.md

```diff
@@ -0,0 +1,279 @@
+---
+title: Support multiple profiles under a single user data folder
+description: How to have different profiles of a WebView2 app running with different profiles under a single user data folder to achieve browsing data separation.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 10/11/2022
+---
+# Support multiple profiles under a single user data folder
+
+To achieve data separation, a WebView2 app can choose to use different user data folders for different WebView2 controls. However, in such way you'll have to be running multiple WebView2 runtime instances (each including a browser process and a bunch of child processes), which means much more consumption for system resources including memory, CPU footprint, disk space, so it is not desirable.
+
+Now we have a more desirable solution: multiple profile support, which enables a WebView2 app to have multiple profiles under a single user data folder, each profile has a dedicated profile folder to save browser data, which means separate browsing data storage, such as cookies, user preference settings, and cached resources. All the WebView2 controls associated with the same profile share the single profile folder. 
+
+Note that the profile maps to the same concept of Edge browser profile.
+
+WebView2 multiple profile API allows you to create and manipulate profiles to work with your WebView2 controls to fulfill your needs.
+
+
+<!-- ====================================================================== -->
+## Specify the profile when creating WebView2
+
+This method creates an options object to give specific information of a profile, including `ProfileName` and `IsInPrivateModeEnabled`. This object is to be used to specify the target profile when creating a WebView2 control.
+
+
+<!-- ------------------------------ -->
+
+# [C#](#tab/csharp)
+
+TODO
+
+
+<!-- ------------------------------ -->
+
+# [C++](#tab/cpp)
+
+[ICoreWebView2Environment10::CreateCoreWebView2ControllerOptions Method](/microsoft-edge/webview2/reference/win32/icorewebview2environment10#createcorewebview2controlleroptions)
+
+[ICoreWebView2ControllerOptions Interface](/microsoft-edge/webview2/reference/win32/icorewebview2controlleroptions)
+
```

Comment:
Consider expanding/updating the API Ref links in the Overview article.  Design this new article in conjunction with updating the Overview article.

The present draft of Multiple Profiles article has API Ref links that should be added to the top-level Overview of APIs article such as maybe in "User data" h2 section:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/overview-features-apis#user-data

List of Win32 API Ref types/members that are in new Multiple Profiles article (so far, in PR 2260) but not in Overview article:
* ICoreWebView2ControllerOptions interface
* ICoreWebView2Environment10 interface - maybe add type-level link for Win32 only, b/c Intro content exists there.
* ICoreWebView2_13::Profile property (get)
* ICoreWebView2Profile - the Overview article lists only ICoreWebView2Profile::DefaultDownloadFolderPath property - decide whether new article should add some specific member links for that type (such as DefaultDownloadFolderPath) & if so, copy into Overview article.

Reply:
Thanks Michael! Yep, I agree Multiple Profile support should have a place in the Overview article, how about we create a separate section under the "User data" section?

Reply:
fixed in my PR #2260 https://github.com/MicrosoftDocs/edge-developer/pull/2260 that merges into this PR: Added section "Multiple profiles" in "Overview of APIs" article, under the "User data" section.